raster/r.what/main.c

```diff
@@ -527,6 +634,11 @@ int main(int argc, char *argv[])
         cache_hit = cache_miss = 0;
     }
 
+    if (strcmp(opt.format->answer, "json") == 0) {
+        fprintf(stdout, "%s\n", json_serialize_to_string(root_value));
```

Comment:
We could have an option for pretty print or use it as the default.

```c
char *serialized_string = NULL;
serialized_string = json_serialize_to_string_pretty(root_value);
if (serialized_string == NULL)  {
   G_fatal_error(_("Failed to initialize pretty JSON string."));
)
// fprintf(stdout, "%s\n", json_serialize_to_string(root_value));
puts(serialized_string);
json_free_serialized_string(serialized_string);
```

Reply:
As default I would say, adding additional option is probably unnecessary.

Reply:
Will update.

Reply:
Done.